plugins/AudioFileProcessor/AudioFileProcessorWaveView.cpp

```diff
@@ -376,14 +382,16 @@ void AudioFileProcessorWaveView::zoom(const bool out)
 void AudioFileProcessorWaveView::slide(int px)
 {
 	const double fact = qAbs(double(px) / width());
-	auto step = range() * fact * (px > 0 ? -1 : 1);
+	double step = range() * fact * (px > 0 ? 1 : -1);
```

Comment:
why this inversion?

Reply:
Because I changed how it works. I didn't know how it was supposed to work and it wasn't working, so I redesigned it slightly.

Reply:
I would ask opinion from @sakertooth since you mentioned a redesign

Reply:
It's fine to leave `auto` here. `fact` is a double, so `step` will be a `double`. For the actual inversion, if the function changed but still achieves the same goal (i.e., zooms properly), it is fine. I will try to do some testing if time permits.

Reply:
Could we change this back to using `auto`? The inversion itself is fine if the behavior ultimately changed for the better.